docs/calling-the-asset-delivery-api.md

```diff
@@ -0,0 +1,168 @@
+# Calling the Asset Delivery APIs
+
+## SDK methods
+
+The [Asset Delivery
+API](https://www.sharetribe.com/api-reference/asset-delivery-api.html) allows
+retrieving the data of a single asset or a set of assets by alias or by version.
+The SDK has corresponding methods for each case.
+
+### assetByAlias()
+
+The `assetByAlias()` method retrieves data for a single asset given a version
+alias. It takes an object as its argument and the object must contain:
+
+* `alias`: currently must always have the value `latest`
+* `path`: the path of the asset
+
+For example:
+
+```js
+const promise = sdk.assetByAlias({
+  alias: "latest",
+  path: "content/translations.json"
+});
+```
+
+### assetByVersion()
+
+The `assetByVersion()` method retrieves data for a single asset given a version
+string. It takes an object as its argument and the object must contain:
+
+* `version`: the version string
+* `path`: the path of the asset
+
+For example:
+
+```js
+const promise = sdk.assetByAlias({
+  version: "ESz8ULLX68PiENQV",
+  path: "content/translations.json"
+});
+```
+
+### assetsByAlias()
+
+The `assetsByAlias()` method retrieves data for a set of assets using the
```

Comment:
Comment: Maybe clarify here that the results are not in the same order as `paths`.